introspection/server.go

```diff
@@ -36,7 +36,7 @@ const (
 // exposing Clair metrics and traces
 type Server struct {
 	// configuration provided when starting Clair
-	conf config.Config
+	conf *config.Config
```

Comment:
What is the reason for this change? Just saving the allocation when passing?

Reply:
Yeah, I noticed keeping the same signature would mean either two copies (`clair.serveAPI` not being passed a pointer, then `introspection.New` not getting a pointer) or using the explicit dereference syntax for no apparent gain.